docs/concepts/glossary.md

```diff
@@ -0,0 +1,140 @@
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+#### Flink Application Cluster
+
+A Flink Application Cluster is a dedicated [Flink Cluster](./glossary#flink-cluster) that only
+executes a single [Flink Job](./glossary#flink-job). The lifetime of the
+[Flink Cluster](./glossary#flink-cluster) is bound to the lifetime of the Flink Job. Formerly
+Flink Application Clusters were also known as Flink Clusters in *job mode*.
+
+#### Flink Cluster
+
+The distributed system consisting of (typically) one Flink Master process and one or more Flink
+Taskmanagers processes.
+
+#### Event
+
+An event is a statement of fact about a change of the state of the domain modelled by the
+application. Events can be input and/or output of a stream or batch processing application.
+Events are special types of [records](./glossary#Record)
+
+#### ExecutionGraph
+
+see [Physical Graph](./glossary#physical-graph)
+
+#### Function
+
+Functions, or user-defined functions (UDFs), are implemented by the user and encapsulate the
+application logic of a Flink program. Most Functions are wrapped by a corresponding
+[Operator](./glossary#operator).
+
+#### Instance
+
+The term *instance* is used to describe a specific instance of a specific type (usually
+[Operator](./glossary#operator) or [Function](./glossary#function)) during runtime. As Apache Flink
+is mostly written in Java, this corresponds to the definition of *Instance* or *Object* in Java.
+In the context of Apache Flink, the term *parallel instance* is also frequently used to emphasizes
+that multiple instances of the same [Operator](./glossary#operator) or
+[Function](./glossary#function) type are running in parallel.
+
+#### Flink Job
+
+A Flink Job is the runtime representation of a Flink program. A Flink Job can be either submitted
+to a long running [Flink Session Cluster](./glossary#flink-session-cluster) or it can be started as a
+self-contained [Flink Application Cluster](./glossary#flink-application-cluster).
+
+#### JobGraph
+
+see [Logical Graph](./glossary#logical-graph)
+
+#### Logical Graph
+
+A logical graph is a directed graph describing the high-level logic of a stream processing program.
+The nodes are [Operators](./glossary#operator) and the edges indicate input/output-relationships or
+data streams or data sets.
+
+#### Managed State
+
+Managed State describes application state, which has been registered with the framework. For
+Managed State Apache Flink will take care about persistence and rescaling among other things.
```

Comment:
"For Managed State,..." (here we need the comma)

Reply:
Done